src/document.h

```diff
@@ -545,6 +545,16 @@ struct Document {
         dc.SetBackground(wxBrush(wxColor(Background())));
         dc.Clear();
         if (!rootgrid) return;
+        loopv(i, sys->imagelist) sys->imagelist[i]->trefc = 0;
+        rootgrid->ImageRefCount();

```

Comment:
Why is this happening on every draw? better to do it at the end of loading

Reply:
Ah sorry. I changed that.